cmd/root.go

```diff
@@ -25,8 +26,12 @@ func NewRootCmd() *cobra.Command {
 		Use: "hanko",
 	}
 	cmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file")
-	initConfig()
-	err := initPersister()
+	err := initConfig()
+	if err != nil {
+		_, _ = fmt.Fprintln(os.Stderr, err)
```

Comment:
Should we switch to log.Fatal in the init error cases?

Reply:
Yeah, it might me more meaningful and easier to read than `fmt.Fprintln; os.Exit(1)`.